docs/pages/debugging.md

```diff
@@ -0,0 +1,26 @@
+Debugging ArrayFire Issues {#debugging}
+===============================================================================
+
+Some general advice for debugging mysterious issues with ArrayFire follows.
+
+First, try running with additional debugging information:
```

Comment:
Keeping in mind that this document will be further modified to add more information, we should divide this into sections.

Instead of the first two lines, just a heading that says "Using Environment Variables" is sufficient to explain the section and main agenda of this document - debugging.

Reply:
Fixed.